mcp-server-quill/scripts/start-server.sh

```
@@ -0,0 +1,49 @@
+# --- Main Script ---
+check_requirements
+setup_env
+
+echo "Starting Quill MCP Server..."
+echo "Server will be available at: http://localhost:8001"
+echo "API docs: http://localhost:8001/docs"
+echo "MCP endpoint: http://localhost:8001/mcp/"
+echo ""
+
+# Start the server using uv
+cd "$PROJECT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)